views/machines/index.html

```diff
@@ -74,31 +87,53 @@
 <div class="row">
     <div class="col-md-12">
         <h1 class="page-header">Machines 
-            <button id="newMachine" type="button" class="btn btn-green pull-right" onclick="createVMdialog()" 
-            {% if cloudPlatform == "openstack" %}disabled=""{% endif %}>        
-            <span class="glyphicon glyphicon-plus" style="font-size:15px"></span>&nbsp;&nbsp;New Machine</button>
+
+        <span id="projectChoiceBlock" class="pull-right">
+            Current Project:
+            <select style="height:34px" id="projectChoice" class="btn btn-default btn-xs" onchange="makeAjaxCalls()"></select>
+           <!-- <div id="loading-project" class="loader"></div> -->
+        </span>
+
         </h1>
         <div id="error" class="alert alert-danger text-center" role="alert" style="display:none">
             <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span> <span id="errormessage"></span>
             <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
         </div>
        
-        <div style="height: 34px">
-        <span id="projectChoiceBlock">
-            Current Project:
-            <select style="height:34px" id="projectChoice" class="btn btn-default btn-xs" onchange="makeAjaxCalls()"></select>
-            <span id="loadingStatus"></span>
-        </span>
-
-        <span id="SearchBlock" style="float:right; height:auto;">
-<!--
-            Search:
-            Ideally the search box should go here but i can't seem find it anywhere in this repo
--->
-            <button type="button" class="btn btn-default" id="filter-button" title="Table Filters" onclick="filterTableDialog()"><span class="glyphicon glyphicon-cog"></span></button>
+<div style="height: 34px">
+        <button id="newMachine" type="button" class="btn btn-green" onclick="createVMdialog()" title="Create New Machine"
+        {% if cloudPlatform == "openstack" %}disabled=""{% endif %}>
```

Comment:
This should probably be indented and the closing angle bracket dropped onto a newline to make it clear this is all one HTML tag.